src/types/fastlane.ts

```diff
@@ -97,4 +97,55 @@ export interface IFastlaneInstance {
     };
     FastlanePaymentComponent: (options: IFastlaneComponentOptions) => Promise<IFastlanePaymentComponent>;
     FastlaneCardComponent: (options: Omit<IFastlaneComponentOptions, 'shippingAddress'>) => IFastlaneCardComponent;
+}
+
+export interface IFastlaneOptions {
+    shippingAddressOptions: IFastlaneAddressOptions,
+    cardOptions: IFastlaneCardOptions,
+    styles: IFastlaneStyleOptions
+}
+
+export interface IFastlaneAddressOptions {
+    // default: empty array = all locations allowed
+    allowedLocations: [string];
```

Comment:
This type syntax describes a field that takes an array with only one element that is of "type" `string`. Is this correct or is this field an array of "type" `string`?

```suggestion
    allowedLocations: string[];
```

Reply:
updated.